canary/webrtc-c/jobs/runner.groovy

```diff
@@ -272,8 +262,28 @@ pipeline {
         string(name: 'GIT_HASH')
         booleanParam(name: 'FIRST_ITERATION', defaultValue: true)
     }
+    
+    // Set the role ARN to environment to avoid string interpolation to follow Jenkins security guidelines.
+    environment {
+        AWS_KVS_STS_ROLE_ARN = credentials('CANARY_STS_ROLE_ARN')
+    }
```

Comment:
Let's review and check if we can add this into the subsequent  PRs.

  `CREDENTIALS_BINDING = 'true' # https://www.jenkins.io/doc/pipeline/steps/credentials-binding/ `